instrumentation/opentelemetry-instrumentation-aio-pika/src/opentelemetry/instrumentation/aio_pika/instrumented_exchange.py

```diff
@@ -0,0 +1,52 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+
+import aiormq
+from aio_pika import Exchange, RobustExchange
+from aio_pika.abc import AbstractMessage
+
+from opentelemetry import propagate, trace
+from opentelemetry.instrumentation.aio_pika.span_builder import SpanBuilder
+from opentelemetry.trace import Span
+
+
+class InstrumentedExchange(Exchange):
```

Comment:
Make these classes private?

Reply:
they are not private, they are imported in the instrumentor